jans-cedarling/bindings/cedarling_python/example.py

```diff
@@ -60,20 +60,23 @@
 # field resource_type and id is mandatory
 # other fields are attributes of the resource.
 resource = ResourceData(resource_type="Jans::Issue",
-                        id="random_id", org_id="some_long_id")
+                        id="random_id", org_id="some_long_id", country="US")
 # or we can init resource using dict
 resource = ResourceData.from_dict({
     "type": "Jans::Issue",
     "id": "random_id",
-    "org_id": "some_long_id"
+    "org_id": "some_long_id",
+    "country": "US"
 })
 
```

Comment:
Please add the decoded form of each token as a comment. Like so:
```python
"""
{
    "sub": "..."
}
"""
action_token = ...
```

Reply:
Added in [commit](https://github.com/JanssenProject/jans/pull/9776/commits/abf5f3a4b2c5b7b4a00e6af8409986d6cf7056f7)